woocommerce/woocommerce-ios
# [Mobile Payments] UI for payment gateway selection

Closes: #6993
Part of: #6992



### Description


This PR includes the new payment gateway selection screen to be used when both WCPay and Stripe are active on a site.
The logic to use this screen is coming up in #7059 and #6992, but I'm trying to keep PRs small, so this one is just the UI.

### Testing instructions


The screen is not used yet in code. I would suggest using SwiftUI previews, but I've been having trouble with them today.
You can also check out #7059 instead and go to Settings > In-Person Payments on a US store that has both WCPay and Stripe active.

### Screenshots


No gateway selected:

Light|Dark
-|-
![Screen Shot 2022-06-09 at 16 36 00](https://user-images.githubusercontent.com/8739/172873579-4d293125-5839-440c-869c-a25f9c11e675.png)|![Screen Shot 2022-06-09 at 16 36 14](https://user-images.githubusercontent.com/8739/172873593-d589da81-baa5-490e-a70b-93f6f0ffa9d7.png)

WCPay selected

Light|Dark
-|-
![Screen Shot 2022-06-09 at 16 36 20](https://user-images.githubusercontent.com/8739/172873602-dcfd0aaa-1dca-48e4-9140-dbffa9829786.png)|![Screen Shot 2022-06-09 at 16 36 24](https://user-images.githubusercontent.com/8739/172873604-163e8126-6820-41a6-9422-f14f78c0095c.png)

Stripe selected

Light|Dark
-|-
![Screen Shot 2022-06-09 at 16 36 29](https://user-images.githubusercontent.com/8739/172873610-bd7d943e-f27e-40c3-a9ff-b56d271644c9.png)|![Screen Shot 2022-06-09 at 16 36 34](https://user-images.githubusercontent.com/8739/172873614-41c5450c-b7ce-4909-a4e2-0428c9cf7257.png)


---
